### DISCUSSION TITLE
Current mass

### DISCUSSION THREAD

**Discussion by DanRRRR** [epochpic/epoch#565] [General]
Short question: is current%mass in the code relativistic or non-relativistic ?

```
current => species_list(ispecies)%attached_list%head
**amass = current%mass**

**Comment by Status-Mirror** [comment]
`current%mass` refers to the rest mass of a real particle within the macro-particle. For example, a macro-electron will have `current%mass = 9.1093837e-31`, regardless of the macro-particle weight.

**Comment by DanRRRR** [comment]
Thanks!
